src/components/link/link.html.twig

```diff
@@ -8,6 +8,7 @@
         type: '' (can be '', 'standalone', 'cta', 'primary', 'secondary')
         inverted: false (boolean),
         no_visited: false (boolean),
+        button_behaviour: false (boolean),
```

Comment:
I would prefer to keep this as an exception in the site header (just adding the button classes there), and not a real variant of the link
This is a hacky thing, and we should not make it too visible to users